src/assets/sass/scss/objects/_buttons.scss

```diff
@@ -200,10 +207,37 @@
   align-items: center;
   border: 1px solid $offwhite;
   display: flex;
-  height: 5rem;
   justify-content: center;
   width: 30rem;
 
+  button {
+    background: $eu-jade;
+    border: 1px solid $eu-jade;
+    margin-left: 1rem;
+    transition: background-color 0.5s ease;
+
+    &:hover,
+    &.active {
+      background: $white;
+      color: $eu-jade;
+    }
+
+    &:active,
+    &:focus {
+      background: $eu-jade;
+      box-shadow: none;
+      color: $white;
+    }
+  }
+
+  &.btns-set-height-3em {
```

Comment:
i would prefer to use padding instead of an explicit height. that would work better if the button height would change or the content would wrap. the same comment applies to btns-set-height-5em. futhermore it is a bit strange that the class is called "3em" but the height is "3rem". that is confusing. i would prefer a more descriptive name, but that might be difficult to come up with. perhaps "btns-set-medium-padding" / "btns-set-small-padding".

Reply:
Removed those classes